theano/sandbox/rng_mrg.py

```diff
@@ -1518,6 +1521,7 @@ def multinomial_wo_replacement(self, size=None, n=1, pvals=None,
         warnings.warn('MRG_RandomStreams.multinomial_wo_replacement() is '
                       'deprecated and will be removed in the next release of '
                       'Theano. Please use MRG_RandomStreams.choice() instead.')
+        assert size is None
```

Comment:
The definition of size for multinomial_wo_replacement and choice aren't the same. size must be None for multinomial_wo_replacement. But the size of choice correspond to the n parameter of multinomial_wo_replacement.